mbedtls-sys/build/cmake.rs

```diff
@@ -46,26 +46,18 @@ impl super::BuildConfig {
             cmk.define("CMAKE_TRY_COMPILE_TARGET_TYPE", "STATIC_LIBRARY");
         }
 
-        let mut dst = cmk.build();
+        let dst = cmk.build();
 
-        dst.push("lib");
```

Comment:
Why? This change breaks Windows.

Reply:
I was using the `dst` in a second place, so removed the `push()`. Now it is fixed by using `join()` instead. Is it compatible with Windows? Since tests passed I couldn't validate this part.